### PR TITLE
build: more README updates

### PR DESCRIPTION
**Pull request by ae6xe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,20 +5,21 @@ http://www.arednmesh.org
...
 
 Amateur Radio frequencies are relatively clean of noise from the commercial
 allocations and ensure usability for Amateur Radio Operators.  This firmware
 enables 802.11n wireless networks to be created and expanded with minimal 
 to no pre-planning or IT expertise.  A user can deploy a 'node' anywhere
 to connect in and extend an AREDN network.  Device hardware options exist to
-provide sector coverage, build point-to-point links, and enable end users
-to connect in.  High speed link rates are routinely achieved over long 
-distances, e.g. 60Mbps+ on 10MHz channels over 80km links. 
+provide sector coverage, build point-to-point links, and connecting end
```
Comment (dman776) [inline README.md]: .... and connect end....

Comment (ae6xe) [reply README.md]: corrected.

**Comment by dman776** [comment]
Should we include a reference to the ARRL site on how to acquire a Tech license?

**Comment by ae6xe** [comment]
good idea.  added at bottom the reference.

**Review by dman776 — APPROVED** [review]
looks good

**Pull request merged**